tests/client/test_v1.py

```diff
@@ -0,0 +1,24 @@
+"""Tests for Shaarli REST API v1 client"""
+from unittest import mock
+
+from shaarli_client.client import ShaarliV1Client
+
+SHAARLI_URL = 'http://domain.tld/shaarli/'
+SHAARLI_SECRET = 's3kr37!'
+
+
+def test_constructor():
+    """Instantiate a new client"""
+    ShaarliV1Client(SHAARLI_URL, SHAARLI_SECRET)
+
+
+@mock.patch('requests.request')
```

Comment:
Yup, you're very likely to see a lot of them in upcoming test code mixed with pytest fixtures!

It's worth noticing that [decoration order matters](https://stackoverflow.com/questions/22677280/checking-call-order-across-multiple-mocks), e.g. here with 2 mocks and the [`tmpdir`](http://docs.pytest.org/en/latest/tmpdir.html) fixture:

```python
@mock.patch('module2.function2')
@mock.patch('module1.function1')
def test_some_stuff(fun1, fun2, tmpdir):
    """Test using multiple mock decorators and the tmpdir pytest fixture"""
    [...]
```